python/setup.cfg

```diff
@@ -35,22 +35,17 @@ opendp =
 	py.typed
 
 [options.extras_require]
-# If this changes, update the smoke-test matrix.
 numpy = 
 	numpy>=1.17,<2.0
 	randomgen
 scikit-learn = 
 	scikit-learn
 	%(numpy)s
-polars =
-	# A strict version requirement is necessary to keep the serialization format of LazyFrames stable.
-	# If you change the version of Polars, 
-	# be sure to also change the Polars version in rust/Cargo.toml and test binary compatibility.
```

Comment:
Is there a reason to delete the comments? My sense is that these are still true, and they point out changes that aren't obvious and would need to be made across multiple files.

Reply:
I see what happened: these comments are not preserved by ConfigParser. I swapped to another package ConfigUpdater.